SAMv1.tex

```diff
@@ -559,8 +570,8 @@ \section{Recommended Practice for the SAM Format}
   \begin{enumerate}[label=\arabic*]
   \item When one segment is present in multiple lines to represent a multiple
 	mapping of the segment, only one of these records should have the secondary
```

Comment:
Again, the current specs are missing clarification as to what's allowed when the primary alignment is chimeric -
 you can have multiple non-secondary lines, but only one of these can be non-supplementary.